Sources/Player/Player.swift

```diff
@@ -122,6 +122,10 @@ public extension Player {
 			return nil
 		}
 
+		guard let offlineStorage = Player.initializedOfflineStorage() else {
```

Comment:
[question] we already have log for this one, right?

Reply:
Yes, internally we do a `PlayerLoggable.withDefaultDatabase(error: error)` if we fail to initialize the Database.